ground/gcs/src/plugins/boards_brotronics/Brotronics.pluginspec

```diff
@@ -0,0 +1,12 @@
+<plugin name="Brotronics" version="1.0.0" compatVersion="1.0.0">
+    <vendor>Tau Labs</vendor>
+    <copyright>(C) 2012-2015 Tau Labs</copyright>
```

Comment:
This one is tricky.  I'm leaving the vendor string along to be consistent with all other plugins.  Also appending ourselves to the copyright.